src/App.jsx

```diff
@@ -1,34 +1,19 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
+import { useState, useEffect } from 'react'
 import './App.css'
+import Header from './components/Header'
+import Articles from './components/Articles'
+import Searchbar from './components/Searchbar'
 
 function App() {
-  const [count, setCount] = useState(0)
+
+  const [articles, setArticles] = useState([])
```

Comment:
Why would this state be here and passing the info on props? There is no data, nor an api call here. It isn't needed.